qemu/tests/cfg/guest_iommu_test.cfg

```diff
@@ -38,3 +38,16 @@
             only Linux
             enable_guest_iommu = yes
             guest_iommu_option = pt
+    variants diff_parameter:
+        - eim_off:
+            iommu_eim = off
+            check_key_words = "DMAR-IR: dmar0 does not support EIM;"
+            check_key_words += "DMAR-IR: Enabled IRQ remapping in xapic mode"
```

Comment:
Applying: iommu: add new scenarios with different parameters
.git/rebase-apply/patch:22: trailing whitespace.
            check_key_words += "DMAR-IR: Enabled IRQ remapping in x2apic mode"

Please delete the whitespace at the end of the line

Reply:
@XueqiangWei updated, thanks.